db/config.cc

```diff
@@ -533,6 +533,8 @@ db::config::config(std::shared_ptr<db::extensions> exts)
         "Throttles all streaming file transfer between the data centers. This setting allows throttles streaming throughput betweens data centers in addition to throttling all network stream traffic as configured with stream_throughput_outbound_megabits_per_sec.")
     , stream_io_throughput_mb_per_sec(this, "stream_io_throughput_mb_per_sec", liveness::LiveUpdate, value_status::Used, 0,
         "Throttles streaming I/O to the specified total throughput (in MiBs/s) across the entire system. Streaming I/O includes the one performed by repair and both RBNO and legacy topology operations such as adding or removing a node. Setting the value to 0 disables stream throttling")
+    , stream_plan_ranges_batch_percentage(this, "stream_plan_ranges_batch_percentage", liveness::LiveUpdate, value_status::Used, 10,
+        "Specify the percentage of ranges to stream in a single stream plan")
```

Comment:
1. nit: s/ragnes/ranges/ in commit message

2. `stream_plan_ranges_batch_percentage` seems unclear.  Given its description, how about `stream_plan_ranges_percentage`?

Reply:
OK. The word batch is not going to help much. I will change to the short one as you suggested.

Reply:
Should use a fraction (float 0..1), not percentage, gives better control.

Reply:
It gives the granularity of 0.01 already. I am not sure why the a more finer granularity like 0.001 or 0.0001 is needed. In practice, user will probably just use the value of 100%  if they ever want to change the default value of stream_plan_ranges_batch_percentage.

Reply:
It's more consistent with other fractions. See reduce_cache_sizes_at, wasm_memory_cache_fraction,  background_writer_scheduling_quota, dynamic_snitch_badness_threshold, background_writer_scheduling_quota.

Reply:
OK. I am convinced by the consistent argument. Changed in v2. Thanks.